frontend/js/src/utils/utils.tsx

```diff
@@ -619,7 +620,8 @@ const getAlbumArtFromListenMetadata = async (
   // directly access additional_info.release_mbid instead of using getReleaseMBID because we only want
   // to query CAA for user submitted mbids.
   const userSubmittedReleaseMBID =
-    listen.track_metadata?.additional_info?.release_mbid;
+    listen.track_metadata?.additional_info?.release_mbid ||
+    listen.track_metadata?.release_mbid;
```

Comment:
I don't know if this field is used, but including it as a fallback for user-submitted MBIDs just in case.